spec/jobs/pusher_activity_notification_job_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe PusherActivityNotificationJob, '#perform' do
```

Comment:
Do you think this provides much value beyond what happens in the feature spec already?